spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestDataSourceOptions.java

```diff
@@ -219,12 +219,14 @@ public void testSplitOptionsOverridesTableProperties() throws IOException {
             .option(SparkReadOptions.SPLIT_SIZE, String.valueOf(splitSize))
             .load(tableLocation);
 
-    Assert.assertEquals("Spark partitions should match", 2, resultDf.javaRDD().getNumPartitions());
+    assertThat(2)
```

Comment:
this should be the other way around: `assertThat(resultDf.javaRDD().getNumPartitions())...isEqualTo(2)`